src/Data/Tree/Binary/Properties.agda

```diff
@@ -48,3 +51,14 @@ private
 map-id : ∀ (t : Tree N L) → map id id t ≡ t
 map-id (leaf x)     = refl
 map-id (node l v r) = cong₂ (flip node v) (map-id l) (map-id r)
+
+map-compose : ∀ {f₁ : N₁ → N₂} {f₂ : N → N₁} {g₁ : L₁ → L₂} {g₂ : L → L₁} (t : Tree N L) → map (f₁ ∘ f₂) (g₁ ∘ g₂) t ≡ map f₁ g₁ (map f₂ g₂ t)
```

Comment:
Could you introduce a line break in the type? I'd leave all of the implicit arguments
on the first line & then have the main statement on a separate line.

Cf. [the style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md#file-structure) for how we typically indent things.